.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,22 @@
+name: Lint Pull Requests
+
+on: 
+  pull_request:
+    branches:
+    - '*'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: pip install flake8 lintly
```

Comment:
This won't work on Python 3.3. You should probably use the previously developed "deps.py".

Reply:
You mean the dev folder? I interpreted it as scripting folder used to run travis-ci only and thus obsolete by moving to git workflows.

Linting sources locally can easily be achieved by just running `flake8 --config=tox.ini` via command line, which can be installed via `pip install flake8`. 

Is there more happening in the `dev` package, which I am not aware of?

Reply:
It makes it possible for users developing it locally with a Python 3.3 install to easy get the properly packages to lint, and run the linting rules. Since Python 3.3 is unsupported by a bunch of packages, you need specific versions, and pip even has issues on various CIs I used with various Python packages I develop.

The idea is that we'll probably add AppVeyor and CircleCI to make sure tests are being run on all platforms.

Reply:
> The idea is that we'll probably add AppVeyor and CircleCI to make sure tests are being run on all platforms.

You can run windows and macOS jobs with github workflows.

Reply:
> The idea is that we'll probably add AppVeyor and CircleCI

Which renders the whole attempt here useless, so we can close it.

Reply:
> You can run windows and macOS jobs with github workflows.

The downside being GitHub only supports very new releases, and we've definitely hit OS release-specific issues.